packages/@okta/vuepress-site/docs/guides/protect-your-api/main/springboot/configcors.md

```diff
@@ -1,30 +1,21 @@
-In order to configure CORS in Spring Security you must enable it in the `WebSecurityConfigurerAdapter` you defined in the previous step:
+1. To configure CORS in Spring Security, enable it in the `WebSecurityConfigurerAdapter` that you defined in the previous step:
 
 ```java
-import org.springframework.context.annotation.Configuration;
-import org.springframework.security.config.annotation.web.builders.HttpSecurity;
-import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
-
-@Configuration
-class OktaOAuth2WebSecurityConfigurerAdapter extends WebSecurityConfigurerAdapter {
-
-    @Override
-    protected void configure(HttpSecurity http) throws Exception {
-        // previous configuration
-        ...
-        http.cors();
-    }
+@Override
+protected void configure(HttpSecurity http) throws Exception {
+  // previous configuration
+  ...
+  http.cors();
 }
-``` 
+```
 
-Then configure individual controllers with `CrossOrigin` annotation. For example:
+2. Configure individual controllers with `CrossOrigin` annotation. For example:
 
 ```java
+…
 @CrossOrigin(origins = "http://example.com:80")
-@GetMapping("/hello")
-public String hello() {
-    return "Hello simple example";
-}
+@GetMapping("/api/whoami")
+…
 ```
 
-For more details, refer to the [Spring CORS guide](https://spring.io/guides/gs/rest-service-cors/).
+See the [Spring CORS guide](https://spring.io/guides/gs/rest-service-cors/).
```

Comment:
```suggestion
See the [Spring CORS guide](https://spring.io/guides/gs/rest-service-cors/)
```
Trailing periods for external links render funny:
<img width="459" alt="image" src="https://user-images.githubusercontent.com/99954/155762250-3f079f41-faf6-496e-ada2-edbf95477608.png">

Reply:
What is the desired look? If it's just CSS that needs some tweaking, I might be able to help.